switchmap/poller/snmp/snmp_manager.py

```
@@ -477,466 +504,468 @@ def query(
+                    )
+                    log.log2info(1209, log_message)
+                else:
+                    log_message = (
+                        f"Async SNMP error for {self._poll.hostname}: "
+                        f"{exception_error}"
+                    )
+                    log.log2die(1003, log_message)
+                # Ensure formatted_result is set for exception cases
+                formatted_result = {}
+
+            except Exception as exception_error:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)